.travis/build-deploy.sh

```diff
@@ -37,6 +37,11 @@ if travis_isPullRequest; then
 
     log_info "This is a pull-request build"
     ./mvnw verify $MVN_BUILD_FLAGS
+    (
+        set +e
+        log_info "Running danger"
+        bundle exec danger --verbose
```

Comment:
This needs to be without "bundle exec" now.
But I can't tell you, why it didn't work, when you used "bundle install" earlier - all the gems have been installed according to the logs...

Reply:
@adangel This seems to be a problem with pmdtester, because there is no such file as pmdtester.rb in the lib directory of pmdtester, I now temporarily solve this issue with `require 'pmdtester/runner'`